src/test/java/seedu/eduke8/command/QuizCommandTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.eduke8.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.eduke8.Eduke8Test;
+import seedu.eduke8.exception.Eduke8Exception;
+import seedu.eduke8.question.QuizQuestionsManager;
+import seedu.eduke8.quiz.SingleTopicQuiz;
+import seedu.eduke8.topic.Topic;
+import seedu.eduke8.topic.TopicList;
+import seedu.eduke8.ui.Ui;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+public class QuizCommandTest extends Eduke8Test {
+
+    @Test
+    public void execute_badTopicName_exceptionThrown() {
+        Ui ui = new Ui();
+        TopicList topicList = createTestTopicList();
+        Command badQuizCommand = new QuizCommand(topicList, 1, "badname", ui);
+        Eduke8Exception exception = assertThrows(Eduke8Exception.class, () -> {
+            topicList.find("badname");
+        });
```

Comment:
I think for this you can call execute on the command and then just assertTrue(true) which represents that the the execute doesn't enter a loop since a bad topic was given

Reply:
Understood! And done!!